examples/ansible/setup_libvirt_host.yml

```diff
@@ -1,129 +1,120 @@
-          become: true
-          ansible.builtin.systemd_service:
-            name: "{{ workload.service.name }}"
-            enabled: true
-          when:
-            - workload_service_start is success
-
-        - name: Wait for service to be ready for workload {{ workload.name }}
-          when:
-            - workload.service.wait_for is defined
-          block:
-
-            - name: Check service path for workload {{ workload.name }}
-              ansible.builtin.wait_for:
-                path: "{{ workload.service.wait_for.path }}"
-              when:
-                - workload.service.wait_for.path is defined
-
-            - name: Check service port for workload {{ workload.name }}
-              ansible.builtin.wait_for:
-                port: "{{ workload.service.wait_for.port }}"
-              when:
-                - workload.service.wait_for.port is defined
-
-#
-# Verify basic operation of the Libvirt service
-#
-
-- name: Verify that Libvirt is operational
-  hosts: alphost
-  tasks:
-
-    - name: Query running Libvirt VMs
-      become: true
-      community.libvirt.virt:
-        command: list_vms
-      register: virt_list_vms
+    - name: Install tools for kvm-client
+      ansible.builtin.command: >-
+        podman container runlabel install registry.opensuse.org/suse/alp/workloads/tumbleweed_containerfiles/suse/alp/workloads/kvm-client:latest
+      register: workload_runlabel_install
+      changed_when:
+        - ('already exist' not in workload_runlabel_install.stdout)
 
-    - name: Report running Libvirt VMs, if any
+    - name: Ensure libvirtd is stopped and disabled
+      ansible.builtin.service:
+        name: "{{ item }}"
+        state: stopped
+        enabled: false
+      loop:
+        - libvirtd.service
+        - libvirtd-ro.socket
+        - libvirtd-admin.socket
+        - libvirtd-tcp.socket
+        - libvirtd-tls.socket
+      register: service_result
+      failed_when: >
+        service_result is failed and
+        ("Could not find the requested service" not in service_result.msg)
+
+    - name: Ensure kvm-server-container.service is started and enabled
+      ansible.builtin.service:
+        name: kvm-server-container.service
+        state: started
+        enabled: true
+      notify: Reload systemd
+
+    - name: Reload systemd right now if necessary
+      ansible.builtin.meta: flush_handlers
+
+    - name: Enable and start log and lock drivers
+      ansible.builtin.service:
+        name: "{{ item }}"
+        state: started
+        enabled: true
+      loop:
+        - container-virtlogd.service
+        - virtlogd.socket
+        - virtlogd-admin.socket
+        - container-virtlockd.service
+        - virtlockd.socket
+        - virtlockd-admin.socket
+
+    - name: Enable and start other drivers
+      ansible.builtin.service:
+        name: "container-virt{{ item }}d.service"
+        state: started
+        enabled: true
+      loop:
```

Comment:
As above about moving lists like this to a more obvious location for ease of maintenance purposes.